### PR TITLE
Add comment section that stores data in DataStore. Displays to About page.

### PR DESCRIPTION
**Pull request by saraholiviawu**
Add comments section that stores comments and displays publicly regardless of refreshing page.

Video shows 
1) Addition of comment
2) Refreshing of page to demonstrate data subsists
https://drive.google.com/file/d/1gG8d78-1en5Xmt9SaGm-Vhb-ki8-1nwD/view

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -28,23 +36,47 @@
...
+        long id = entity.getKey().getId();
+        String name = (String) entity.getProperty("name");
+        String text = (String) entity.getProperty("text");
+        long timestamp = (long) entity.getProperty("timestamp");
+
+        Comment singleComment = new Comment(id, name, text, timestamp);
+        comments.add(singleComment);
+    }
+    Gson gson = new Gson();
+    // String json = new Gson().toJson(list);
```
Comment (sabadas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: remove

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -52,14 +52,39 @@ function createListElement(text) {
...
+
+function createCommentElement(comment) {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comment';
+
+  const nameElement = document.createElement('h4');
+  nameElement.innerText = comment.name;
+
+//   const breakElement = document.createElement('BR');
+
```
Comment (sabadas) [inline portfolio/src/main/webapp/script.js:71]: remove

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -52,14 +52,39 @@ function createListElement(text) {
...
+  deleteButtonElement.addEventListener('click', () => {
+    deleteComment(comment);
+
+    // Remove the task from the DOM.
+    commentElement.remove();
+  });
+
+  
+  commentElement.appendChild(nameElement);
+//   commentElement.appendChild(breakElement);
```
Comment (sabadas) [inline portfolio/src/main/webapp/script.js:86]: remove

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -28,23 +36,47 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  List<String> list = new ArrayList<>();
+//   List<String> list = new ArrayList<>();
```
Comment (sabadas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:39]: remove

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (saraholiviawu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fixed

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (saraholiviawu) [reply portfolio/src/main/webapp/script.js]: Fixed

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (saraholiviawu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fixed

**Pull request merged**